### ISSUE TITLE
Implement polychord and diffusive nested sampling

### ISSUE THREAD

**Issue by ben18785** [pints-team/pints#305]

**Comment by MichaelClerx** [comment]
Before we do this, could we have a look at the nested sampling interfaces together?

**Comment by ben18785** [comment]
Yep! Are you about today? Happy to come over. Alternatively Tuesday/Wednesday next week? Sorry for the deluge of samplers — just using issues as a way of keeping track of ideas.

**Comment by MichaelClerx** [comment]
Diffusive nested sampling already has ticket #152 

Nested sampling interface in general has tickets #107 and #130
